src/api/github/org.test.ts

```diff
@@ -0,0 +1,66 @@
+import { createAppAuth } from '@octokit/auth-app';
+
+import { GETSENTRY_ORG } from '@/config';
+import { ClientType } from '@api/github/clientType';
+import { getClient } from '@api/github/getClient';
+import { OctokitWithRetries as octokitClass } from '@api/github/octokitWithRetries';
+
+import { GitHubOrg } from './org';
+
+describe('constructor', function () {
+  beforeAll(function () {
+    octokitClass.mockClear();
+    new GitHubOrg('cheese please', {});
+  });
+
+  it('is instantiated once', function () {
+    expect(octokitClass).toHaveBeenCalledTimes(1);
+  });
+
+  it('is instantiated with userToken', function () {
+    expect(octokitClass).toHaveBeenCalledWith({ auth: 'cheese please' });
+  });
+
+  it('throws if no user token', function () {
+    expect(() => {
+      new GitHubOrg('', {});
+    }).toThrow('No user token, try setting GH_USER_TOKEN.');
+  });
+
+  it('does not try to get an org installation', function () {
+    expect(octokitClass.apps.getOrgInstallation).toHaveBeenCalledTimes(0);
+  });
+});
+
+describe('bindAPI', function () {
+  beforeAll(async function () {
+    const org = await new GitHubOrg('cheese please', {
```

Comment:
does this need `await`?

Reply:
maybe some weird behavior related async/await is causing the error in CI?

Reply:
Simplified in 691ce4a51faee940342adc9951d5b318aa465c12.